### PR TITLE
Add OMFIT to affiliated pkgs

### PR DESCRIPTION
**Pull request by rocco8773**
This adds OMFIT to the affiliated packages page, <https://www.plasmapy.org/affiliates>.

@smithsp It appears that your repo link <https://github.com/gafusion/OMFIT-source/> does not work.  I'm guessing this is because the repo is private.  Would you like me to remove the link or keep it?

The page currently renders like...

image

### CODE REVIEW

File: data/affiliated_pkgs.json [hunk 1]
```diff
@@ -44,5 +44,28 @@
...
+      "home": "https://omfit.io",
+      "repo": "https://github.com/gafusion/OMFIT-source/",
+      "image": "/images/affiliated_pkg_logos/OMFIT_logo.gif"
+    },
+    "description": {
+      "short": "A open-source Python 3 package for plasma physics.",
+      "long": "OMFIT - One Modeling Framework for Integrated Tasks - is an integrated modeling  and experimental data analysis framework written in python.  The goal of OMFIT is to enhance existing scientific workflows and enable new integrated modeling capabilities."
+    },
+    "affiliated": false,
+    "keywords": ["plasma", "physics", "Python", "Python 2", "Python 3", "fusion", "simulation", "diagnostics"],
```
Comment (smithsp) [inline data/affiliated_pkgs.json:63]:
```suggestion
    "keywords": ["plasma", "physics", "Python", "Python 3", "fusion", "simulation", "diagnostics"],
```

File: data/affiliated_pkgs.json [hunk 2]
```diff
@@ -44,5 +44,28 @@
     },
     "status": "stable",
     "license": "MIT"
+  },
+  "omfit": {
+    "maintainer": {
+      "name": null,
```
Comment (smithsp) [inline data/affiliated_pkgs.json:50]:
```suggestion
      "name": Sterling Smith,
```

File: data/affiliated_pkgs.json [hunk 3]
```diff
@@ -44,5 +44,28 @@
...
+      "contact": null
```
Comment (smithsp) [inline data/affiliated_pkgs.json:51]:
```suggestion
      "contact": smithsp@fusion.gat.com
```

File: data/affiliated_pkgs.json [hunk 4]
```diff
@@ -44,5 +44,28 @@
...
+      "long": "OMFIT - One Modeling Framework for Integrated Tasks - is an integrated modeling  and experimental data analysis framework written in python.  The goal of OMFIT is to enhance existing scientific workflows and enable new integrated modeling capabilities."
+    },
+    "affiliated": false,
+    "keywords": ["plasma", "physics", "Python", "Python 2", "Python 3", "fusion", "simulation", "diagnostics"],
+    "dates": {
+      "added": "2021 June 22",
+      "last_updated": "2021 June 22"
+    },
+    "status": "stable",
+    "license": "User Agreement"
```
Comment (smithsp) [inline data/affiliated_pkgs.json:69]:
```suggestion
    "license": "User Agreement" https://form.omfit.io
```

**Review by smithsp — APPROVED** [review]
@rocco8773 Thanks for this addition.  I have made some minor suggestions/changes.

**Comment by rocco8773** [comment]
@smithsp I've made the updates you requested, with a little modification to the get the user agreement link to work.  The entry now looks like...  And I've confirmed the user agreement link works properly.

image

**Comment by smithsp** [comment]
@rocco8773 Looks great.  Thanks for your help.

**Review by smithsp — APPROVED** [review]

**Pull request merged**